### ISSUE TITLE
Can this code achieve the accuracy of the paper?

### ISSUE THREAD

**Issue by jingong** [kenziyuliu/DGNN-PyTorch#13]
Can this code achieve the accuracy of the paper?

**Comment by kenziyuliu** [comment]
I've only tried once to repro the spatial stream and was able to get about 85% (see uploaded model in `runs/` folder). Due to hardware limits and the fact that certain hyperparams were missing, I have not tried further tuning

**Comment by jet-yangqs** [comment]
Which data set do you used to get 85% test result? I tested your uploaded pretrained model. I can get about 94% on ntu rgb+d cross view val data preprocessed same as 2s-AGCN.
